lib/matplotlib/backend_tools.py

```diff
@@ -26,10 +27,9 @@
 _log = logging.getLogger(__name__)
 
 
-class Cursors(object):
-    """Simple namespace for cursor reference"""
```

Comment:
I would still write a docstring, as the default is `An enumeration.` right now.

Reply:
sure, done